src/DetailsView/components/details-view-command-bar.tsx

```diff
@@ -87,6 +90,14 @@ export class DetailsViewCommandBar extends React.Component<DetailsViewCommandBar
         );
     }
 
+    private renderFarItems(): JSX.Element {
+        if (this.props.narrowModeStatus.isCommandBarCollapsed) {
+            return this.renderCommandButtonsMenu();
```

Comment:
It's great that it's so easy to render this differently depending on the size of the window!